scripts/upload_gatk_docs/upload_gatk_tools_docs.py

```diff
@@ -0,0 +1,238 @@
+                            # Discard all lines up to and including the stop marker
+                            while index_beginning[1] not in line:
+                                line = next(lines)
+                            line = next(lines)
+                    else:
+                        # Starts at the beginning and skips line until the end of the cut
+                        if beginning[0] in line:
+                            # Discard all lines up to and including the stop marker
+                            while beginning[1] not in line:
+                                line = next(lines)
+                            line = next(lines)
+
+                        # Get the title of article (i.e. GATK tool name)
+                        if '<h1>' in line:  # e.g.    <h1>CNNVariantCaller</h1>
+                            article_title = line.split('<h1>')[1]
+                            article_title = article_title.split('</h1>')[0]
+
+                            # Remove '**EXPERIMENTAL** ' and '**BETA** ' from titles
+                            # e.g. '**BETA** Title'.split('**') -> ['','BETA',' Title']
+                            if '**' in article_title:
+                                article_title = article_title.split('**')[2].lstrip(' ') + ' (' + article_title.split('**')[1] + ')'
+
+                    # Filter out line with PHP code
+                    if '?>' not in line and '<h1' not in line and ending[0] not in line:
+                        if 'class="accordion-body collapse"' in line:
+                            line = line.replace('class="accordion-body collapse"', '')
+
+                        # Add clean HTML to file
+                        clean_lines.append(line)
+
+                    # Starts at the beginning of the end that need to be cut and skips line until the end of the cut
+                    if ending[0] in line:
+                        # Discard all lines up to and including the stop marker
+                        while ending[1] not in line:
+                            line = next(lines)
+                        line = next(lines)
+            except StopIteration:
+                # Create the dictionary of updated gatkdoc
+                article_dict[copied_file] = GatkDocs(title=article_title,
+                                                     file_name=copied_file,
+                                                     local_path=full_path)
+
+        # Write to a clean gatk doc
+        with open(full_path, 'w') as clean_file:
+            clean_file_data = "".join(clean_lines)
+            clean_file_soup = bs(clean_file_data)
+            format_html = clean_file_soup.prettify()
+            clean_file.write(format_html)
+
+    # Output files that was updated
+    print(str(len(copied_files)) + ' files copied and cleaned up')
+
+    # Return GATK Doc Dict
+    return article_dict
+
+
+def post_section(gatk_version, username, token, headers):
```

Comment:
is this function actually posting a section or simply retrieving the url needed to post an article inside that section? or both? i think we said this last time but this function might need a more descriptive name